### PR TITLE
Fixed up proptypes and props for contributors page/components..

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Some more cleanup for the contributors page. I didn't realize #228 got merged already, so this is additional work related to that.

## Related Tickets and Documents

#229
#228

## What gif best describes this PR or how it makes you feel?

Christopher Walken sweeping a hallway

### CODE REVIEW

File: pages/contributors.js [hunk 1]
```diff
@@ -143,47 +143,21 @@ export default function Contributors({ contributors }) {
...
-      updated_at: PropTypes.string.isRequired,
     })
   ).isRequired,
 }
 
 export async function getStaticProps() {
   if (!process.env.GITHUB_TOKEN) {
     throw new Error(
-      `Missing the environment variable GITHUB_TOKEN. If this is happening for local development, ensure you've generated a token and included it in your .env file, e.g. GITHUB_TOKEN="some-token". To generate a personal access token, follow the steps in the GitHub documentation, https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token.`
+      `Missing the environment variable GITHUB_TOKEN. If this is happening for local development, ensure you've generated a token and included it in your .env file, e.g. GITHUB_TOKEN="some-token". To generate a personal access token, follow the steps in the GitHub documentation, https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token . For the scope of the token being created, select public_repo only from within the repo scope.`
```
Comment (nickytonline) [inline pages/contributors.js:160]: Made the error message more clear @sethburtonhall as you weren't sure which scope to add when creating the GitHub token.

File: components/contributors/IndividualContributor.js [hunk 2]
```diff
@@ -122,24 +122,15 @@ const IndividualContributor = ({ props }) => {
...
-
-IndividualContributor.defaultProps = {
-  name: '',
-  login: '',
-  twitter_username: '',
-  avatar_url: '',
-  blog: '',
-  html_url: '',
-  contributions: '',
+  contributor: PropTypes.shape({
```
Comment (nickytonline) [inline components/contributors/IndividualContributor.js:125]: The original list is all the properties in the object, but we never use them all, so refining the prop types for a contributor to the properties we render.

File: components/contributors/ContributorsList.js [hunk 3]
```diff
@@ -28,7 +28,17 @@ const ContributorsList = ({ contributors, isFounders }) => {
 }
 
 ContributorsList.propTypes = {
-  contributors: PropTypes.array.isRequired,
+  contributors: PropTypes.arrayOf(
```
Comment (nickytonline) [inline components/contributors/ContributorsList.js:31]: The original list is all the properties in the object, but we never use them all, so refining the prop types for a contributor to the properties we render.

**Comment by Cerchie** [comment]
Awesome!

**Pull request merged**